internal/helpers/template.go

```diff
@@ -21,8 +21,14 @@ import (
 	"github.com/hashicorp/vault-secrets-operator/internal/template"
 )
 
-// regexCache provides a global LRU cache holding compiled regexes.
-var regexCache *lru.Cache[string, *regexp.Regexp]
+var (
+	// RenderOptionExcludeRaw sets the global sync option for controlling the exclusion
+	// of _raw from the destination secret.
+	// This is usually set from main via the command line arg --global-rendering-options
+	RenderOptionExcludeRaw bool
```

Comment:
For this specific case, with the code as it currently is, this looks ok, but I would love to avoid mutable package-level variables as a general pattern. As we find more use-cases for passing flag config down through to templating functions, it would be great to establish a pattern that's more sustainable and safe. Maybe a stateful template helper struct that gets passed through to the clients that will consume it?